p4_symbolic/ir/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains build rules for bmv2 json protobuf
```

Comment:
By the way, there is typically no need for such a comment in a BUILD file because by convention it will declare targets for all things in that folder, so the comment is redundant. A comment would only be needed if the BUILD files does something out of the ordinary.